src/vic3/effect_validation.rs

```diff
@@ -564,13 +564,20 @@ pub fn validate_create_pop(
 ) {
     // This effect is undocumented
 
+    fn sum_fractions_warner<E: ErrorLoc>(sum_fractions: i64, loc: E) {
+        if sum_fractions != 100_000 {
+            let msg = format!(
+                "fractions should add to exactly 1, currently {}.{}",
```

Comment:
You should zero-pad the fraction here, otherwise 0.5 and 0.05 will both print as 0.5.